lib/config.js

```diff
@@ -0,0 +1,28 @@
+const {app, BrowserWindow, dialog} = require('electron');
+const Configfile = require('electron-config');
+const configfile = new Configfile();
+const path = require('path');
+
+/**
+ * Handles the selection of the directories.
+ * @param {Object} current_window            - The window object of which to start the dialog from
+ * @param {Boolean} startup                  - To determinate wheather
+ * @return {folder}                          - Returns the folder
+ */
+function selectFolders(current_window, startup) {
```

Comment:
This should be in window-settings. There is no need to create a new module to wrap electron-config. In this specific case, the window-settings is the one that has to worry about the interface of selecting the folders and to use electron-config to store those values.